components/Footer/index.js

```diff
@@ -8,7 +8,9 @@ class Footer extends React.Component {
   render(): ?ReactElement {
     return (
       <footer className="Footer">
-        <p className="Footer-legal">Ⓒ 2015 Skookum. All Rights Reserved.</p>
+        <div className="InnerMax">
```

Comment:
InnerMax is already all over the site and working. I will keep moving on other items and will come back and manually apply inside each component later. Cool?